### ISSUE TITLE
Link a usable app from Readme.md

### ISSUE THREAD

**Issue by aadrian** [sprang/Inkpad#160]
Please link from the main page - Readme.md a usable app, something the users can still install, e.g. https://itunes.apple.com/app/inkpad-vector-design-illustration/id1057007769

**Comment by alistairmcmillan** [comment]
Good idea.

Could add a list of all the Inkpad forks that follow the simple rules that Steve released his source under (i.e. everyone that has made their source available and made non-trivial improvements to their forks). 

What does everyone think?

**Comment by JanX2** [comment]
Yes. This is a good idea! Maybe we should name forks that don’t adhere to the rules as well. Just not link to them. ;)

**Comment by alistairmcmillan** [comment]
Personally I'd like to stay positive. :)

Also, naming forks that don't adhere to the rules seems like a full time job for a team of twelve. :)

For what it's worth I've been talking to a few people who have forks in the App Store. I'm putting together a simple little review of each, that focusses a lot on how they are adhering to the rules. I'll link it here once it's posted for everyone's amusement.

**Comment by JanX2** [comment]
@alistairmcmillan That’s great. Thank you!
